static/css/live-tail.css

```diff
@@ -0,0 +1,15 @@
+.query-language-btn,
+.index-btn,
+.live-tail-btn,
+.filter-input {
+    margin: 0;
+    text-decoration: none;
+    height: 40px;
+}
+.log-opt-single-btn{
```

Comment:
why is this css change needed for live tail? can we not use siglens.css

Reply:
It is better to separate the css file of each page from the css file of the project, otherwise the css file of the project will become larger and larger, which will affect the style definition of new functions later. Just my suggest

Reply:
Lets not separate css files, this will help reduce duplicate code.